src/providers/OrganizatioProvider.js

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from 'react'
+import { useApps, useOrganization, usePermissions } from '@aragon/connect-react'
+
+const OrganizationContext = React.createContext()
+
+function OrganizationProvider({ children }) {
```

Comment:
Getting that data from aragon connect is great, although I would prefer to not expose them directly in the provider, but transform them so the old app code works perfectly without requiring any change.